code/datums/jobs/job/survivor.dm

```diff
@@ -1,3 +1,50 @@
+// ***************************************
+// *********** New Survivors
+// ***************************************
+/datum/job/new_survivor
```

Comment:
Why are you making an entirely different typepath?
Survivors aren't used at all right now.